app/assets/scripts/components/charts/bar-chart.js

```diff
@@ -112,6 +112,10 @@ const BarColumnChart = React.createClass({
     let yLabels = yScale.ticks(3)
     let rectWidth = xScale.bandwidth()
 
+    var yLabelRounded = yLabels.map(function (o) {
+      return Math.round(o * 10000) / 10000
```

Comment:
This should be a const, but it also seems like a good candidate for an inline arrow since it isn't reused, i.e.
```
<Axis
  scale={yScale}
  labels={yLabels.map((o) => Math.round(o * 10000) / 10000)}
  orientation='left'
  height={height}
  width={width}
  margin={margin}
/>
```